plato/draw/vispy/Box.py

```diff
@@ -1,11 +1,84 @@
+import functools
+import numpy as np
 from ... import draw
 from .internal import GLPrimitive, GLShapeDecorator
 from .Lines import Lines
+from .Spheres import Spheres
 
-@GLShapeDecorator
-class Box(draw.Box, Lines):
-    __doc__ = draw.Box.__doc__
+def _getter(self, name):
+    return getattr(self._spheres_prim, name)
+
+def _setter(self, value, name):
+    setattr(self._lines_prim, name, value)
+    setattr(self._spheres_prim, name, value)
+
+class Box(draw.Box, GLPrimitive):
```

Comment:
In the vispy backend, is there some way to control the ordering / overlaps of the line and sphere primitives? The shading looks a little funny. (This is probably hard, so no worries if you don't want to change it, @klarh.)
![image](https://user-images.githubusercontent.com/3943761/64751416-0b633e80-d4ea-11e9-8642-56d6f4357807.png)